src/objects/list/demo/columns.twig

```diff
@@ -0,0 +1,15 @@
+{% if columns and columns > 1 and columnsBreakpoint and columnsBreakpoint != 'none' %}
```

Comment:
Is it intentional that `columnsBreakpoint` is required? Is there no use case where someone would want columns on by default?

Reply:
How would someone use multiple breakpoints?

```
o-list--2-columns o-list--3-columns@xl
```

Reply:
@tylersticka Would something like this work better?

```
{% if columns and columns > 1 %}
  {% set _list_class = 'o-list--' ~ columns ~ '-column' %}
{% elseif columns and columns > 1 and columnsBreakpoint %}
  {% set _list_class = 'o-list--' ~ columns ~ '-column' ~ columnsBreakpoint %}
{% else %}
  {% set _list_class = '' %}
{% endif %}
```

Reply:
That would alleviate some of the problems, but if you wanted to do something like this:

```
o-list--2-columns@s  o-list--3-columns@m
```

I think it would prevent that?

(This is one of the reasons I've just been using a `class` property for defining the class attribute in some of my other patterns. It gets complicated pretty quickly!)

Reply:
> (This is one of the reasons I've just been using a class property for defining the class attribute in some of my other patterns. It gets complicated pretty quickly!)

Good idea! 👍 🙂

Reply:
Something like this?
```
args={{
 class: 'o-list--2-column@s o-list--3-column@m'
}}
```